src/clarity-angular/loading/loading-listener.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
```

Comment:
Not a big deal but I thought we were updating copyrights to 2016-2017 when we came across them.